subprojects/native/src/main/java/org/gradle/internal/nativeintegration/services/NativeServices.java

```diff
@@ -101,7 +101,7 @@ public static synchronized void initialize(File userHomeDir, boolean initializeA
                     try {
                         net.rubygrapefruit.platform.Native.init(nativeBaseDir);
                     } catch (NativeIntegrationUnavailableException ex) {
-                        LOGGER.debug("Native-platform is not available.");
+                        LOGGER.debug("Native-platform is not available.", ex);
```

Comment:
Does the stacktrace add some valueable information for end-users here?

Reply:
We log the exception for the `file-events` loading already, and it was needed for me to figure out what was wrong. The cause explained that the library was found but it was of the wrong architecture.

In general I think it's a strong code smell if you catch an exception, log something, but not log the exception.